doc/source/complex_composites.rst

```diff
@@ -0,0 +1,120 @@
+Complex Composites
+==================
+
+Many composites in |project| take advantage of various corrections or
+adjustments to produce the best looking imagery possible. The below
+sections describe the corrections and other related topics used in
+|project|. See the various :doc:`readers/index` documentation for more
+information on what products are available and descriptions of what
+corrections are used.
+
+.. _explain_rgb_composite:
+
+RGB Images
+----------
+
+An RGB image is an image constructed of three different color channels,
+Red, Green, and Blue. By interpreting the values from each channel and
+combining them we can produce an array of colors. This differs from a
+basic grayscale or "luminance" image where the single "L" channel
+determines how white or black a pixel is. For more information on the
+RGB color model see
+`this Wikipedia article <https://en.wikipedia.org/wiki/RGB_color_model>`_.
+
+You may also see RGBA images where the A stands for "Alpha". This Alpha
+channel will determine the opaque or transparency of an image. This is
+typically used in |project| to make invalid or missing data values
+transparent (completely opaque or completely transparent).
+
+.. _sunz_correction:
+
+Solar Zenith Angle Modification
+-------------------------------
+
+Visible channel data is not truly a "reflectance" until it has been adjusted
+for the amount of sun reflecting off of the atmosphere. While this is a very
```

Comment:
Isn't it the incident radiation that needs to be accounted for?

Reply:
I don't know. Someone please correct me (@kathys?).